pkg/users/account-details.js

```diff
@@ -355,3 +347,126 @@ export function AccountDetails({ accounts, groups, shadow, current_user, user })
         </Page>
     );
 }
+
+export const AccountGroupsSelect = ({ name, loggedIn, groups, setError }) => {
+    const [isOpenGroup, setIsOpenGroup] = useState(false);
+    const [selected, setSelected] = useState();
+    const [primaryGroupName, setPrimaryGroupName] = useState();
+    const [history, setHistory] = useState([]);
+    const previousValue = useRef(null);
+
+    useEffect(() => {
+        const usedGroups = groups.filter(group => group.userlist.includes(name));
+        const primaryGroup = groups.find(group => group.userlistPrimary.includes(name));
+        const _primaryGroupName = primaryGroup ? primaryGroup.name : undefined;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17996-20221213-101148-20bf9fbb-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)